cypress/fixtures/markdown/markdown_inline_images_3.html

```diff
@@ -1,2 +1,2 @@
 <h3 class="markdown__heading">In-line Images</h3><p>GIF Image:
-<img alt="gif" class="markdown-inline-img" src="http://localhost:8065/api/v4/image?url=http%3A%2F%2Fi.giphy.com%2FxNrM4cGJ8u3ao.gif"></p>
+<img alt="gif" class="markdown-inline-img" src="http://localhost:8065/api/v4/image?url=http%3A%2F%2Fi.giphy.com%2FxNrM4cGJ8u3ao.gif" style=""></p>
```

Comment:
Just for my own curiosity as I'm not familiar with how Cypress works, what is the reason for the blank `style=""` tag?

Reply:
@deanwhillier This is just the html that gets displayed when we post the corresponding markdown text. We're just, very literally, going "does the html match what the app is displaying". So we must have changed how this markdown gets translated to html.